### ISSUE TITLE
Request: flood fill function - `fillc(x, y, color)`

### ISSUE THREAD

**Issue by eggbanana173** [nesbox/TIC-80#2022]
SPANISH:
mi sugerencia es una función llamada "fillc(x, y, color)"
esta función rellenaría la pantalla de un color definido en una posición especifica como si fuera la herramienta de Paint "rellenar"
sin eliminar las figuras como lo hace "cls(color)"
Ejemplo:
```lua
-- este hipotetico ejemplo hace que rellene a un triangulo que es un "trib()"

function TIC()
 cls()
 trib(25,25,45,80,80,50,4)
 fillc(50,50,4)
end`
```

**Comment by darltrash** [comment]
Translation:

My suggestion is a new function like "fillc(x, y, color)"
Which would fill the entire continuous space of a certain color, like the "fill" feature in MS Paint.
Ejemplo:

```lua
-- This hypothetical example fills a triangle created by "trib()"

function TIC()
 cls()
 trib(25,25,45,80,80,50,4)
 fillc(50,50,4)
end
```

**Comment by joshgoebel** [comment]
Personally I think this would make a great function for an external library. :-)

**Comment by nesbox** [comment]
Hmm, I think it's out of scope of the current api,  and @joshgoebel is right that this would make a great function for an external library

**Comment by anescient** [comment]
I've implemented this in #2683
